### PR TITLE
(maint) Add CODEOWNERS files

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add CODEOWNERS file, so that we can track future changes to files.

## Motivation and Context

This is an attempt to ensure that any changes to our dependencies, whether that is:

* changes to NuGet packages through packages.config files
* updates to custom assemblies which are not tracked through NuGet packages
* updates to custom executables/assemblies which are added directly to the Chocolatey CLI Chocolatey Packages

Are tracked and updated properly.  By making these files/folder owned by the new @chocolatey-credits-reviewers group, it means that a member of that group will be flagged on any PR's that we raised that contain these files.

## Testing

N/A

### Operating Systems Testing

N/A

## Change Types Made

* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist

* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [ ] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v3 compatibility checked?

## Related Issue

N/A

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+packages.config                      @chocolatey-credits-reviewers
+CREDITS.md                           @chocolatey-credits-reviewers
+/src/chocolatey.resources/tools/     @chocolatey-credits-reviewers
+/lib/                                @chocolatey-credits-reviewers
```
Comment (AdmiringWorm) [inline .github/CODEOWNERS:4]: This is referencing users, not teams.

These should probably be

```suggestion
packages.config                      @chocolatey/chocolatey-credits-reviewers
CREDITS.md                           @chocolatey/chocolatey-credits-reviewers
/src/chocolatey.resources/tools/     @chocolatey/chocolatey-credits-reviewers
/lib/                                @chocolatey/chocolatey-credits-reviewers
```

Also, the team needs to be created and have write permissions to this repository for it to work.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (gep13) [reply .github/CODEOWNERS]: Ah, I didn't know it needed to be fully declared like that.  Yes, the group already exists, and I will verify permissions just now.

**Review by AdmiringWorm — APPROVED** [review]
The changes LGTM, however I would be hesitant to merge this until the following is resolved:

> make sure the team @chocolatey/chocolatey-credits-reviewers exists, is publicly visible, and has write access to the repository

**Comment by AdmiringWorm** [comment]
@gep13 thanks for getting this added.

To verify it is working as expected, it may be best to perhaps open a PR that changes one of the files.

If you open the PR, it should request a review from one of the other members

**Pull request merged**